plugins/oauth/web_client/stylesheets/oauthLoginView.styl

```diff
@@ -101,3 +101,16 @@
 
   .g-oauth-button-icon
     border-right-color darken($brandColor, 30%)
+
+.g-oauth-button-box
+  $brandColor = #205081
```

Comment:
Doesn't look like it. I'll try to find their official brand color and change that.